### ISSUE TITLE
Refactor "compilation"

### ISSUE THREAD

**Issue by amosonn** [circleofcircles/circleofcircles#4]
Seems a bit long, and mixes compilation in general, duality of talking about something and acting it or being it, and substitution of self in general statements (which is sort of the meeting point of these two). They are quite mixed all along, so requires a careful rewrite, not just paragraph shuffling.

**Comment by simsim314** [comment]
לא בטוח כמה אלו דברים שונים. קומפילציה של רעיון לתודעה האנושית היא בהגדרה: הכנסת העצמי בתוך רעיון מופשט. כשאתה פועל לפי רעיון מסויים, זה בגלל שה"אני" התחיל "להאמין" שהרעיון הזה הוא "אמיתי" והוא נכון ביקום שלי ומתחבר אלי. ברור שלפעול לפי הרעיון ולהאמין בו אלו שני דברים שונים, אבל כל פעולה מצריכה מודל מנטלי שבו אני מהווה חלק אינטגרלי ממנו. טוב אני אבדוק.

**Comment by amosonn** [comment]
אני לא בטוח שזה נכון תמיד. לדעתי הכנסתי ה"אני" לתוך המשפט היא רק מקרה פרטי
של קומפילציה של משפט, כי לא כל משפט מדבר על "אני" בתור חלק ממנו. אני מבין
מה אתה אומר שברגע שהמשפט מצטרף ליקום שלך יש פה איזה צעד שכולל אותך, אבל
לדעתי יש קומפילציות שבהן ממש אתה מציב את עצמך, כמו בדוגמה של "אני יושב" או
של "תזהר!", אבל באחרות כמו "יהיה בסדר" או "עדיף לא לצאת לדרך" ההכנסה ליקום
שלך היא בשלבים אחרים של הקומפילציה. וגם הפער בין המשפט הנאמר לבין ההבנה שלך
אותו, והפער בין ההבנה התיאורטית שלך אותו לבין ההבנה החווה שלך אותו, הם שני
פערים שונים.

2018-01-26 11:29 GMT+01:00 simsim314 <notifications@github.com>:

**Comment by simsim314** [comment]
אני מנסה לחלק בין משפט כמו "יש דעה שאומרת שיהיה בסדר". לבין משפט שאומר "אני מאמין שלי יהיה בסדר". 

האני במשפט לא חייב להיות ממש ברמה של הצבה. כל משפט ניתן לנסח בכמה דרכים. לפעמים כשאני אומר לך "אתה בסכנה" עליך להציב את עצמך בתוך "אתה". או כשאני פונה אליך ואומר הזהר עצם הפניה אליך כבר מייצרת "אתה" שלא קיים במילים אלא בכוונה. 

ההנחה היא כמובן שהמשמעות עוברת במשפט הנאמר. אי הבנה או הבנה לקויה של משפט אינני קורא לזה קומפילציה אלא שגיאת סינטקס.

**Comment by simsim314** [comment]
משפט שלא הובן טוב זה לא מפתיע או מייצר פרדוקסים כמו עצם העובדה שכל משפט יש לך אפשרות להזדהות  איתו ולהתחיל להאמין בו ולפעמים אנחנו לא שמים לב שהתחלנו להאמין במשפט או לחילופין מכחישים ומסרבים להאמין במשפט שדוקא כדאי לנו להזדהות איתו. עצם הפרדוקס של מרי מוכיח שיש חוסר הבנה בפילוסופיה של התובנה הזאת

**Comment by simsim314** [comment]
ואתה כמובן מוזמן לקצר או לחדד או לייצר ערך שמדגיש את זה ואולי החפירה שלי תלך לערך של תאוריה מורחבת. צריך גם להוסיף תרגילים לדבר הזה.

**Comment by amosonn** [comment]
טוב, תרגילים צריך להוסיף להכל :). נראה לי שאני לא מצליח להסביר את הנקודה
שלי, אז אני אבצע עריכה ואשלח פי-אר, ואז תקבל או תדחה.

2018-01-26 16:33 GMT+01:00 simsim314 <notifications@github.com>:

**Comment by simsim314** [comment]
קראת את כל ההודעות? יש שלוש, בראשונה הסברתי את הנקודה שלי.

**Comment by amosonn** [comment]
כן, אבל נראה לי שזה רק חלק אחד מהתוכן של הערך שלך. למשל הודגמא של מרי לגבי
ההבדל בין לדבר על לחוות צבע לבין לחוות צבע נראית לי משהו שונה יחסית.

**Comment by simsim314** [comment]
לא זה בדיוק זה בעיני. למרי חסרה היכולת להציב את האני שלה בכל משפטים שמכילים צבע.
